### ISSUE TITLE
[nextcloud] Mariadb pod with persistent storage will not start

### ISSUE THREAD

**Issue by blinkiz** [groundhog2k/helm-charts#179]
Hello. Thanks for nice Nextcloud helm chart.

Am having problem starting mariadb that is bundled with Nextcloud chart. Mariadb starts if not using persistent storage but with '`mariadb.storage.requestedSize: 8Gi`' then it fails.
PV created with openbs cStor seems fine.
```
'2021-01-15  9:21:47 0 [ERROR] mysqld: Got error 'Can't create file' when trying to use aria control file '/var/lib/mysql/aria_log_control'
2021-01-15  9:21:47 0 [ERROR] Plugin 'Aria' init function returned error.
2021-01-15  9:21:47 0 [ERROR] Plugin 'Aria' registration as a STORAGE ENGINE failed.
2021-01-15  9:21:47 0 [ERROR] InnoDB: Operating system error number 13 in a file operation.
2021-01-15  9:21:47 0 [ERROR] InnoDB: The error means mysqld does not have the access rights to the directory.
2021-01-15  9:21:47 0 [ERROR] InnoDB: Operating system error number 13 in a file operation.
2021-01-15  9:21:47 0 [ERROR] InnoDB: The error means mysqld does not have the access rights to the directory.
2021-01-15  9:21:47 0 [ERROR] InnoDB: Cannot open datafile './ibdata1'
2021-01-15  9:21:47 0 [ERROR] InnoDB: Could not open or create the system tablespace. If you tried to add new data files to the system tablespace, and it failed here, you should now edit innodb_data_file_path in my.cnf back to what it was, and remove the new ibdata files InnoDB created in this failed attempt. InnoDB only wrote those files full of zeros, but did not yet use them in any way. But be careful: do not remove old data files which contain your precious data!
2021-01-15  9:21:47 0 [ERROR] InnoDB: Database creation was aborted with error Cannot open a file. You may need to delete the ibdata1 file before trying to start up again.
2021-01-15  9:21:48 0 [ERROR] Plugin 'InnoDB' init function returned error.
2021-01-15  9:21:48 0 [ERROR] Plugin 'InnoDB' registration as a STORAGE ENGINE failed.
2021-01-15  9:21:48 0 [ERROR] Failed to initialize plugins.
2021-01-15  9:21:48 0 [ERROR] Aborting
```

Any ideas how to continue troubleshooting this?

**Comment by nikkoura** [comment]
This is related to #171.
You have to add two lines to your NextCloud values.yaml:
```patch
 mariadb:
   enabled: true
+  podSecurityContext:
+    fsGroup: 999
```

**Comment by blinkiz** [comment]
Yes that worked great.

**Comment by groundhog2k** [comment]
I fixed issue #171 - Thank you for testing/help!
